working dir/widgets/comet/Utils.java

```diff
@@ -489,7 +489,7 @@ static void drawHudCircle(GraphicsContext gc, GameSize field, double x, double y
 		drawHudCircle(gc, field, x, y, r, 0, D360, color);
 	}
 	static void drawHudPolygon(GraphicsContext gc, GameSize field, double[] xs, double[] ys, long pointCount) {
-		throwIfNot(xs.length==ys.length);
+		failIf(xs.length!=ys.length, () -> "Amount of x and y coordinates not equal");
```

Comment:
- 'amount of' is used for uncountable words, use 'number of'
- 'x and y' is little ambiguous about how to apply the and operator

I propose: `Number of x coordinates and y coordinates do not match`.

Reply:
Yes, I think you should add appropriate messages to these fail conditions anyways.